src/create/tokio.rs

```diff
@@ -78,7 +81,36 @@ pub async fn new_path<H, P: AsRef<Path> + Clone>(
 where
   H: Handler<Writer = Compat<WriteHalf<Connection>>> + Send + 'static,
 {
-  let stream = Endpoint::connect(path).await?;
+  let stream = {
+    #[cfg(unix)]
+    {
+      use tokio::net::UnixStream;
+
+      UnixStream::connect(path).await?
+    }
+    #[cfg(windows)]
+    {
+      use std::time::Duration;
+      use tokio::net::windows::named_pipe::ClientOptions;
+      use tokio::time;
+
+      // From windows-sys so we don't have to depend on that for just this constant
+      pub const ERROR_PIPE_BUSY: i32 = 231i32;
```

Comment:
Could you put a link here where the number is coming from?

Reply:
Done